### PR TITLE
bugfix + new-feature: Fix contribute guidelines link and add Dutch translation file.

### PR DESCRIPTION
**Pull request by JxxIT**
# Proposed Changes

> Adds a dutch translation and fixes the contribute guidelines link.

### CODE REVIEW

File: spotify/translations/nl.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    name: Log-niveau
+    description: >-
+      Bepaalt het niveau van logboekdetails die de add-on biedt.
+  name:
+    name: Naam
+    description: >-
+      De naam van je device (het Spotify Connectie-doel), zoals het er
+      uit ziet op de officiële Spotify-cliënten.
+  bitrate:
+    name: Bitsnelheid
```
Comment (RlndVt) [inline spotify/translations/nl.yaml:13]: In the description you call it 'bitrate'. Change the name to match.

File: spotify/translations/nl.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    description: >-
+      De bitrate die Spotify moet gebruiken. Hoe hoger, hoe beter de geluidskwaliteit,
+      de add-on verbruikt echter meer gegevens.
```
Comment (RlndVt) [inline spotify/translations/nl.yaml:16]: '[..] gegevens.' reads strange in my opinion. Maybe call it 'internet'?

"[...] de add-on verbruikt echter meer internet."

**Comment by JxxIT** [comment]
Fixed it

**Comment by JxxIT** [comment]
No! Please, reopen!

**Pull request closed without merging**